DragaliaAPI.Shared/Resources/Export.json

```diff
@@ -0,0 +1,10962 @@
+[
```

Comment:
why do you have CharaStories.json, DragonStories.json, Export.json, and two view jsons? surely you only need the first 2